app/src/main/java/android/coding/interview/makeitawesome/adapter/PhotosAdapter.java

```diff
@@ -27,7 +55,15 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int i) {
 
     @Override
     public void onBindViewHolder(ViewHolder holder, int i) {
-        holder.mText.setText(mData.get(i));
+        holder.mText.setText(mData.get(i).getTitle());
+
+        final ImageView img = holder.mImage;
+        Glide.with(holder.mImage.getContext()).load(mData.get(i).getThumbnailUrl()).asBitmap().into(new SimpleTarget<Bitmap>(80, 80) {
+            @Override
+            public void onResourceReady(Bitmap resource, GlideAnimation glideAnimation) {
```

Comment:
keeping `img` reference here in callback causes multiple loading of different urls into same position on fast scrolling.
Easily fixed by just simple loading with ```.into(holder.mImage)``` without callback.